index.html

```diff
@@ -0,0 +1,193 @@
+          </ul>
+        </div>
+      </div>
+
+      <div class="mobile-bar">
+        <nav class="mobile-navbar">
+          <img class="hamburger" src="./img/menu_icon.png" alt="open menu" />
+        </nav>
+      </div>
+
+      <div class="title-container">
+        <p class="header-phrase">"Hello! Coders"</p>
+        <h1 class="header-title">
+          MODERN REACT <br />
+          GLOBAL SUMMIT <br />
+          2022
+        </h1>
+      </div>
+    </header>
+    <main>
+      <section class="intro-container">
+        <p class="intro-text">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab nisi,
+          debitis nesciunt ea cupiditate minima quae ut illo quas, quam neque
+          dicta consectetur.
+        </p>
+      </section>
+
+      <section class="place-container">
+        <p class="date">2022.12.02(FRI) to 03(SAT)</p>
+        <p class="place">@National Museum of Lima, Convention Center</p>
+      </section>
+
+      <section id="program" class="program-container">
+        <h2 class="program-title">Main Program</h2>
+        <hr class="program-line" />
+
+        <div class="program-items-container">
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/conference_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Lecture</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/exhibition_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Exhibition</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/workshop_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Workshop</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/show_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Show</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+        </div>
+
+        <button type="button" class="join-button">
+          Join React Global Summir 2022
+        </button>
+
+        <a class="program-link" href="#">See the whole program</a>
+      </section>
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. 

- In this section, the template has 5 five cards however in your project there are just 4. Please add one more card. :eyes: :eyes: 

Your design
![image](https://user-images.githubusercontent.com/98363075/203673808-75106439-cec9-40e3-a6e9-717a5e79c199.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203673871-84b50c20-bd76-4318-ae5c-43ca44121b24.png)

- Also please make sure to add the hover effect shown in the template. There is a white border when you hover over every card. :pray: :pray: 

![image](https://user-images.githubusercontent.com/98363075/203673993-3000de7f-de1e-4cb5-a762-f7749f2340c9.png)